### PR TITLE
Created E2E tests for OrderController

### PR DESCRIPTION
**Pull request by MateuszHocholowski**
with some additional changes to pojos

### CODE REVIEW

File: src/test/java/com/orzechazo/eshop/controllers/OrderControllerE2ETest.java [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@ExtendWith(SpringExtension.class)
+@DataJpaTest
+@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
+class OrderControllerE2ETest {
+    @Autowired
+    OrderRepository orderRepository;
```
Comment (RadekKaniecki) [inline src/test/java/com/orzechazo/eshop/controllers/OrderControllerE2ETest.java:40]: Most of those variables can be private

File: src/test/java/com/orzechazo/eshop/controllers/OrderControllerE2ETest.java [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    @Autowired
+    UserRepository userRepository;
+    OrderServiceImpl orderService;
+    UserServiceImpl userService;
+    OrderController orderController;
+    MockMvc mockMvc;
+    private static final String DB_ORDER_ID = BootstrapUsersAndOrders.DB_ORDER_ID1;
```
Comment (RadekKaniecki) [inline src/test/java/com/orzechazo/eshop/controllers/OrderControllerE2ETest.java:47]: Don't assign one constant to another one, whole purpose of static constant is to use them across many classes, just use here ones defined in Bootstrap class

File: src/test/java/com/orzechazo/eshop/controllers/OrderControllerE2ETest.java [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+
+        mockMvc.perform(get("/orders/" + DB_ORDER_ID)
+                .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isOk())
+                .andExpect(jsonPath("$.orderId",equalTo(DB_ORDER_ID)))
+                .andExpect(jsonPath("$.orderDate").value(DB_DATE_TIME))
+                .andExpect(jsonPath("$.admissionDate",equalTo(DB_DATE_TIME)))
+                .andExpect(jsonPath("$.realizationDate", equalTo(DB_DATE_TIME)))
+                .andExpect(jsonPath("$.paymentDate",equalTo(DB_DATE_TIME)))
+                .andExpect(jsonPath("$.totalPrice",is(new BigDecimal("100").intValue())));
```
Comment (RadekKaniecki) [inline src/test/java/com/orzechazo/eshop/controllers/OrderControllerE2ETest.java:89]: Create constant for this price

File: src/test/java/com/orzechazo/eshop/controllers/OrderControllerE2ETest.java [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+    }
+
+    @Test
+    void getOrderByOrderIdThatIsNotInDB() throws Exception{
+        mockMvc.perform(get("/orders/orderIdNotInDB")
```
Comment (RadekKaniecki) [inline src/test/java/com/orzechazo/eshop/controllers/OrderControllerE2ETest.java:94]: orderIdNotInDB to constant

**Comment by MateuszHocholowski** [comment]
Corrected

File: src/test/java/com/orzechazo/eshop/bootstrap/tests/BootstrapUsersAndOrders.java [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+
+        log.debug("Loading Orders...");
+
+        Order order1 = new Order();
+        order1.setOrderId(DB_ORDER_ID1);
+        order1.setOrderDate(DATE_TIME);
+        order1.setAdmissionDate(DATE_TIME);
+        order1.setRealizationDate(DATE_TIME);
+        order1.setPaymentDate(DATE_TIME);
+        order1.setTotalPrice(new BigDecimal("100"));
```
Comment (RadekKaniecki) [inline src/test/java/com/orzechazo/eshop/bootstrap/tests/BootstrapUsersAndOrders.java:46]: ORDER1_TOTAL_PRICE should be created here

**Review by RadekKaniecki — APPROVED** [review]

**Pull request merged**